### PR TITLE
[as2_behavior] ament format, passing all tests

### PR DESCRIPTION
**Pull request by javilinos**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Issue(s) this addresses   |#582 #490|
| ROS2 version tested on | Humble|
| Aerial platform tested on ||

---

## Description of contribution in a few bullet points

* as2_behavior with ament format

### CODE REVIEW

File: as2_behaviors/as2_behavior/tests/basic_server_test.cpp
Comment (miferco97) [inline as2_behaviors/as2_behavior/tests/basic_server_test.cpp:1]: Why is this file deleted ?

**Review by miferco97 — CHANGES_REQUESTED** [review]

File: as2_behaviors/as2_behavior/tests/basic_server_test.cpp
Comment (RPS98) [reply as2_behaviors/as2_behavior/tests/basic_server_test.cpp:1]: I think has change the file name to as2_behavior_gtest.cpp

Comment (javilinos) [reply as2_behaviors/as2_behavior/tests/basic_server_test.cpp:1]: as2_behavior_gtest.cpp contains the code from basic_server_test.cpp that is then tested with google tests method.

**Review by miferco97 — APPROVED** [review]
